crates/opencascade-sys/include/wrapper.hxx

```diff
@@ -104,6 +107,13 @@ typedef opencascade::handle<Poly_Triangulation> Handle_Poly_Triangulation;
 typedef opencascade::handle<TopTools_HSequenceOfShape> Handle_TopTools_HSequenceOfShape;
 typedef opencascade::handle<Law_Function> HandleLawFunction;
 
+typedef opencascade::handle<TColgp_HArray1OfPnt> Handle_TColgpHArray1OfPnt;
+
+inline std::unique_ptr<Handle_TColgpHArray1OfPnt> new_HandleTColgpHArray1OfPnt_from_TColgpHArray1OfPnt(const TColgp_HArray1OfPnt &array) {
+    Handle_TColgpHArray1OfPnt handle = &array;
```

Comment:
This line is pretty sus, probably it's taking a reference to stack memory that gets freed?

Reply:
Good eye! That was indeed it. Had to change the signature to take a `unique_ptr` instead of a bare ref, and now it works.